packages/store/internals/src/object-utils.ts

```diff
@@ -0,0 +1,9 @@
+export class ObjectUtils {
```

Comment:
Maybe just use plain functions here?

```ts
export function merge ....
```

Functions are first class citizens in JS. IMHO there is no point to have a class with static methods.

Reply:
I moved it to a namespace so that it keeps the `ObjectUtils` wrapper but does not create a class.

Reply:
good)